<?xml version="1.0"?>
<launch>
    <arg name="target_frame" default="/base" />
    <arg name="positions" default="/people_tracker/positions" />
    <arg name="marker" default="/people_tracker/marker_array" />
    <arg name="machine" default="localhost" />
    <arg name="user" default="" />


    <node name="rightHandCamSetup" pkg="baxter_tools" type="camera_control.py" required="false" output="screen" args="-o right_hand_camera -r 1280x800"/>

    <node name="leftHandCamSetup" pkg="baxter_tools" type="camera_control.py" required="false" output="screen" args="-o left_hand_camera -r 1280x800"/>

    <!-- start circle detection and tracking for left camera -->
    <node name="LeftHandCircleFinder" pkg="circle_detection" type="circle_finder" required="true" output="screen">
        <param name="camera" type="str" value="left" />
    </node>


    <!-- start circle detection and tracking for right camera -->
    <node name="RightHandCircleFinder" pkg="circle_detection" type="circle_finder" required="true" output="screen">
        <param name="camera" type="str" value="right" />
    </node>

    <!-- view procesed image from left circle tracker  -->
    <node name="view_left" pkg="image_view" type="image_view" required="true">
        <remap from="image" to="/circledetection/left/rgb/processedimage"/>
    </node>

    <!-- view procesed image from right circle tracker  -->
    <node name="view_right" pkg="image_view" type="image_view" required="true">
        <remap from="image" to="/circledetection/right/rgb/processedimage"/>
    </node>
    
    <!-- start bayes people tracker for 3D tracking  -->
    <!-- <include file='$(find bayes_people_tracker)/launch/people_tracker.launch'>
        <arg name='target_frame' value='base' />
    </include>
-->
</launch>